src/unitxt/operators.py

```diff
@@ -770,6 +770,8 @@ def process_value(self, value: Any) -> Any:
 
 
 class AddID(StreamInstanceOperator):
+    """Stores an id value in the designated 'id_field_name' field of the given instance."""
```

Comment:
What kind of id is added?  Worth adding, and showing an example.

Reply:
Operator `Addit` is only used once, in [prepare/metrics/squad.py](https://github.com/IBM/unitxt/blob/5df775d70b0cbb65a1552d4ddf3367475364d145/prepare/metrics/squad.py#L2). with no effect at all. Seems like no effect is expected from it. I suggest to remove class `AddIt` altogether.